packages/atomic/src/components/commerce/atomic-commerce-load-more-products/atomic-commerce-load-more-products.new.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  playExecuteFirstSearch,
+  wrapInCommerceInterface,
+} from '@coveo/atomic/storybookUtils/commerce-interface-wrapper';
+import {parameters} from '@coveo/atomic/storybookUtils/common-meta-parameters';
+import {renderComponent} from '@coveo/atomic/storybookUtils/render-component';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import {html} from 'lit/static-html.js';
+
+const {decorator, play} = wrapInCommerceInterface({skipFirstSearch: true});
+
+const meta: Meta = {
+  component: 'atomic-commerce-load-more-products',
+  title: 'Atomic-Commerce/LoadMoreProducts',
```

Comment:
```suggestion
  title: 'Atomic-Commerce/Interface Components/atomic-commerce-load-more-products',
```
According to https://docs.google.com/document/d/1oXGc9dJqBSQjlw7pLD6eolEuPvCyMUcAlyOhedbr78E/edit?usp=sharing should follow this structure, but I've noticed the tests added for the other commerce components don't follow it. @louis-bompart what should we do about that?